src/components/form/helpers.ts

```diff
@@ -7,3 +7,12 @@ export function unpack(classes?: Classes) {
   const { container = "", input = "", label: lbl = "", error = "" } = _classes;
   return { container, input, lbl, error };
 }
+
+export function dateToFormFormat(date: Date) {
```

Comment:
Don't we wish to display dates in users locale? @SovereignAndrey @ap-justin

Reply:
It would be good to try to display dates formatted to the user's locale(with `.toLocaleDateString()` I think on the Date?), but would need to submit to AWS as standard UTC and expect to receive the same back from AWS too. A simple wrapper function or two could be used to manage that. Maybe try to format to locale and if not valid / set just fallback to `en-GB`?

Reply:
@misicnenad, @SovereignAndrey , `react-hook-form` won't display the date in input field if value is not in this format